index.md

```diff
@@ -36,32 +38,35 @@ This is an overview of all Android versions and their corresponding identifiers
     <td rowspan="2"><i>No data</i></td>
     <td rowspan="2"><i>TBA</i></td>
   </tr>
+
   <tr>
     <td rowspan="2">
       <b><a href="https://developer.android.com/about/versions/12">Android 12</a></b>
     </td>
     <td>
-      Level 32
-      <span class="subversion">Android 12L</span>
-      <sup class="beta">BETA</sup>
+      12L <sup class="beta">BETA</sup>
     </td>
+    <td class="level">Level 32</td>
     <td><code>S_V2</code></td>
     <td rowspan="2">
       Snow Cone
       <sup id="fnref:2"><a href="#fn:2" class="footnote">2</a></sup>
     </td>
   </tr>
+
   <tr>
-    <td>Level 31 <span class="subversion">Android 12</span></td>
+    <td>12</td>
```

Comment:
I would still bring in some of my changes from https://github.com/ebelinski/apilevels/pull/14, because the inconsistency of 4 missing cells probably raises more eyebrows than the missing empty spaces gives wins.